tools/ci/install_datalad

```diff
@@ -7,5 +7,5 @@ sudo apt-get install git-annex-standalone
 # Install datalad system-wide for use with localhost ssh
 sudo apt-get install datalad
 # ... and install it into the virtualenv.
-pip install datalad
+pip install git+https://github.com/datalad/datalad.git
```

Comment:
now that there is a datalad 0.13.0rc1 I think it should be sufficient to declare `~= 0.13.0rc1` for datalad I think.  That would limit though to `0.13` "major" series of datalad.

Reply:
This line will be reverted and setup.py adjusted with the dependency anyway.